units/systemd-resolved.service.in

```diff
@@ -46,6 +46,7 @@ RestrictRealtime=yes
 RestrictSUIDSGID=yes
 RuntimeDirectory=systemd/resolve
 RuntimeDirectoryPreserve=yes
+StandardOutput=journal+console
```

Comment:
Why is this change necessary?

Reply:
In order to make the log messages visible on the terminal.

Reply:
I know what the option does, but why should it be enable for this service by default?

Reply:
Okay, I had made this change for convenience. I will revert it if not required.